devel/onetbb/Portfile

```diff
@@ -85,6 +85,13 @@ if {[string match *gcc* ${configure.compiler}]} {
 }
 
 variant tests description {Enable tests} {
+    configure.pre_args-replace \
+                    -DCMAKE_BUILD_WITH_INSTALL_RPATH:BOOL=ON \
+                    -DCMAKE_BUILD_WITH_INSTALL_RPATH:BOOL=OFF
+
+    configure.args-append   -DTBB_OUTPUT_DIR_BASE="tbb_test_bin"
     configure.args-delete   -DTBB_TEST=OFF
+
+    test.env-append         DYLD_LIBRARY_PATH=${cmake.build_dir}/tbb_test_bin_macports
```

Comment:
There are quite a number of tests being run. Several fail due to dylibs not being found.
Since TBB uses a weird naming for that build directory by default, it is easier to set the specific name to point to it.

Reply:
Setting `CMAKE_BUILD_WITH_INSTALL_RPATH` to `OFF` seems to be sufficient for all tests to pass for me. To fix the malloc issues, can you not simply do `test.env DYLD_LIBRARY_PATH=${prefix}/lib/libgcc`?

Reply:
@mohd-akram I will try your suggestion, thank you.

Reply:
UPD. Yeah, it was an overkill from my side. Your suggestion works just as good.